src/main/java/nl/tudelft/ewi/devhub/server/web/resources/repository/AbstractProjectResource.java

```diff
@@ -196,6 +183,35 @@ public Response showBranchOverview(@Context HttpServletRequest request,
 			} else throw e;
 		}
 
+		switch (branchDeletion) {
```

Comment:
Dont use 1 and -1 for the parameter, but a Boolean instead, and don't put it as parameter when it is null.